### DISCUSSION TITLE
Mutations get stuck in single part

### DISCUSSION THREAD

**Discussion by gulnara** [ClickHouse/ClickHouse#59400] [Q&A]
We run a deletion job on a schedule and periodically we have some mutations get stuck in a single part out of ~1200. Also when we search for that part name (the one that is still in parts_to_do_names) it doesn't exist on the system.parts table. 

Looks like a freeze partition command gets initiated, this is our guess. First the issues happened because we had a backup job run at the same time as our cron job. After staggering the jobs things got unstuck for about 24 hours. However it started happening again and we are still not sure why mutations get stuck. Is there documentation or insight into this? Is the freeze command the issue? So far the way we maned to solve is by killing mutations manually but it's not a sustainable solution.

**Comment by wansanji** [comment]
Do you find the solution in the end?
